html/includes/authentication/ldap.inc.php

```diff
@@ -108,8 +108,9 @@ function get_userlevel($username)
     // Loop the list and find the highest level
     foreach ($entries as $entry) {
         $groupname = $entry['cn'][0];
+        $userlevel = array();
```

Comment:
It shouldn't, the return for get_userlevel and get_user in ldap auth is different from others - check mysql auth for example. Ldap now is brought in line with what the other modules return.

Same for the other comment.